Source/WebSocket/WinRT/winrt_websocket.cpp

```diff
@@ -578,7 +581,8 @@ void MessageWebSocketSendMessage(
 
 HRESULT CALLBACK Internal_HCWebSocketDisconnect(
     _In_ HCWebsocketHandle websocket,
-    _In_ HCWebSocketCloseStatus closeStatus
+    _In_ HCWebSocketCloseStatus closeStatus,
+    _In_opt_ void* context
```

Comment:
does this change not need to be made to winhttp_websocket.cpp as well?

Reply:
I don't think it's actually being build by anyone, so I missed it

Reply:
I believe its used by gsdk, and optionally on win32

---
In reply to: [278250082](https://github.com/Microsoft/libHttpClient/pull/399#discussion_r278250082) [](ancestors = 278250082)